src/renderer/components/theme-settings/theme-settings.js

```diff
@@ -97,17 +97,21 @@ export default defineComponent({
       ]
     },
 
+    /* Themes are devided into 3 groups. The first group contains the default themes. The second group are themes that don't have specific primary and secondary colors. The third group are themes that do have specific primary and secondary colors available. */
     baseThemeNames: function () {
       return [
+        // First group
```

Comment:
If you add these comments then they should be added to `baseThemeValues` too
Maybe add to `colors.js` too

Reply:
`colors.js` is unrelated to the comment. I can add it to `baseThemeValues` but that doesn't sound very useful to me

Reply:
You're right. Added the comment!